modular_citadel/code/modules/jobs/jobs.dm

```diff
@@ -0,0 +1,15 @@
+GLOBAL_LIST_INIT(command_positions, list(
+	"Captain",
+	"Head of Personnel",
+	"Head of Security",
+	"Chief Engineer",
+	"Research Director",
+	"Chief Medical Officer",
+	"Quartermaster"))
+
+GLOBAL_LIST_INIT(supply_positions, list(
```

Comment:
I have no idea what I'm looking at here, but a simple read of this leads me to believe that this is the source of the lack of HoP. You removed it from supply_positions, but didn't put it anywhere else. Perhaps add it to civilian_positions? I don't know if that'll help, but who kows